File: CosmeticaShop.IServices/Models/Brand/BrandModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web;

namespace CosmeticaShop.IServices.Models.Brand
{
    public class BrandModel
    {
        #region [ Свойства ]

        /// <summary>
        /// Ид
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// Наименование
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// Url бренда
        /// </summary>
        public string KeyUrl { get; set; }

        /// <summary>
        /// Видимость
        /// </summary>
        public bool IsActive { get; set; }

        public string SeoKeywords { get; set; }

        public string SeoDescription { get; set; }

        public string PhotoUrl { get; set; }

        public HttpPostedFileBase PhotoFile { get; set; }

        #endregion
    }
}

File: CosmeticaShop.Web/Areas/Admin/assets/js/Form/Brand/BrandModel.js
var Brand = Brand || {};

(function () {

    if (Brand.BrandModel) {
        console.error("Brand.BrandModel уже был создан");
        return;
    }

    /**
     * Класс для описания модели бренда
     * @param {     
     * } theParams 
     * @returns {} 
     */
    Brand.BrandModel = function (theParams) {
        theParams = theParams || {};

        this.Id = ko.observable(theParams.Id || "");
        this.Name = ko.observable(theParams.Name || "");
        this.PhotoUrl = ko.observable(theParams.PhotoUrl || "");
        this.IsActive = ko.observable(theParams.IsActive || false);


        return this;
    };

    /**
     * Определяем конструктор
     */
    Brand.BrandModel.prototype.constructor = Brand.BrandModel;

    Brand.BrandModel.prototype.log = function (text) {
        console.log("Ошибка в классе Brand.BrandModel: " + text);
    };

    Brand.BrandModel.prototype.GetData = function () {
        return {
            Id: this.Id(),
            Name: this.Name(),
            PhotoUrl: this.PhotoUrl(),
            IsActive: this.IsActive()
        }
    }
})();